react/components/StepCounter.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { InjectedIntl } from 'react-intl'
+
+import { Button } from 'vtex.styleguide'
+import { StepCounterControl } from '../typings/typings'
+
+interface Props extends StepCounterControl {
+  intl: InjectedIntl
+  nextDisabled?: boolean
+}
+
+const StepCounter: FC<Props> = ({
+  intl,
+  back,
+  next,
+  nextDisabled,
+  current,
+  total,
+}) => {
+  return (
+    <div className="flex items-center">
+      <div className="c-muted-1 f6">
+        {intl.formatMessage(
+          { id: 'admin/messages.stepCounter' },
+          { current, total }
+        )}
+      </div>
+      <div className="flex-grow-1 tr">
+        {current === 1 ? null : (
```

Comment:
Since you are converting `current` to `string` when passing it to the component tree, that makes the Back Button always render (since `"1" === 1` is `false`). As a result, the user can click back on the first step and go to a blank screen

Reply:
Fixed